GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -247,6 +250,12 @@ void Mesh::generateVAO() {
         glVertexAttribPointer(it->first, 4, GL_FLOAT, 0, 0, 0);
     }
 
+    vaoID_map_[key] = vaoID_;
+	triangle_vboID_map_[key] = triangle_vboID_;
```

Comment:
I may be missing something, but `triangle_vboID_`, `vert_vboID_` and `tex_vboID_` never seem to be set in these diffs.